fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -42,31 +42,65 @@ find_package(fastrtps REQUIRED CONFIG)
 string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastcdr_MAJOR_MINOR_VERSION "${fastcdr_VERSION}")
 string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastrtps_MAJOR_MINOR_VERSION "${fastrtps_VERSION}")
 
+find_library(FastCDR_LIBRARY_RELEASE
+    NAMES fastcdr-${fastcdr_MAJOR_MINOR_VERSION} fastcdr)
+
+find_library(FastCDR_LIBRARY_DEBUG
+    NAMES fastcdr${CMAKE_DEBUG_POSTFIX}-${fastcdr_MAJOR_MINOR_VERSION})
+
+if(FastCDR_LIBRARY_RELEASE)
+    if(FastCDR_LIBRARY_DEBUG)
+        set(FastCDR_LIBRARIES
+            optimized ${FastCDR_LIBRARY_RELEASE}
+            debug ${FastCDR_LIBRARY_DEBUG}
+        )
+    else()
+        set(FastCDR_LIBRARIES
+            ${FastCDR_LIBRARY_RELEASE}
+        )
+    endif()
+else()
+    if(FastCDR_LIBRARY_DEBUG)
+        set(FastCDR_LIBRARIES
+            ${FastCDR_LIBRARY_DEBUG}
+        )
+    endif()
+endif()
```

Comment:
I hope this is what you meant:

https://github.com/ros2/rmw_fastrtps/pull/116/commits/3c34a21082ab8ff9e3c0a3b6d5e3d2a608eab68e